config.json

```diff
@@ -642,9 +642,12 @@
       "slug": "diffie-hellman",
       "core": false,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 5,
       "topics": [
-
+        "Cryptography",
+        "Integers",
+        "Algorithms",
+        "Transforming"
```

Comment:
Yeah I think that belongs at [problem-specifications](https://github.com/exercism/problem-specifications).